src/star.css

```diff
@@ -0,0 +1,25 @@
+.stars__star {
+  list-style: none;
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
It's better to place Block selector at the top of the file. And all Elements should be following after it.